### PR TITLE
PE-726: Excise ArFS tag from data transactions

### PR DESCRIPTION
**Pull request by fedellen**
This PR removes the incorrectly applied ArFS tag from data transactions.

Proposed GitHub release changelog:
- ArFS tag no longer included with data transactions

### CODE REVIEW

**Review by agsuy — APPROVED** [review]
Lgtm!

File: src/arfs/arfsdao.test.ts [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+describe('The ArFSDAO class', () => {
+	const wallet = readJWKFile('./test_wallet.json');
+	const fakeArweave = Arweave.init({
+		host: 'localhost',
+		port: 443,
+		protocol: 'https',
+		timeout: 600000
+	});
+
+	const arfsDao = new ArFSDAO(wallet, fakeArweave, true, 'ArFSDao-Test', '1.0');
```
Comment (arielmelendez) [inline src/arfs/arfsdao.test.ts:20]: Nit: ArFSDao -> ArFSDAO

File: src/arfs/arfsdao.test.ts [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+		const getDecodedTagName = (tag: Tag) => tag.get('name', { decode: true, string: true });
+
+		it('includes the base ArFS tags by default', async () => {
+			const transaction = await arfsDao.prepareArFSObjectTransaction({
+				objectMetaData: stubFileMetaDataTrx,
+				rewardSettings: { reward: W(10) }
+			});
+			expect(transaction.tags.find((tag) => getDecodedTagName(tag) === 'ArFS')).to.exist;
+			expect(transaction.tags.find((tag) => getDecodedTagName(tag) === 'App-Name')).to.exist;
+			expect(transaction.tags.find((tag) => getDecodedTagName(tag) === 'App-Version')).to.exist;
```
Comment (arielmelendez) [inline src/arfs/arfsdao.test.ts:46]: Might also want to check the total tags count.

**Review by arielmelendez — APPROVED** [review]

File: src/arfs/arfsdao.test.ts (hunk 1 above)
Comment (fedellen) [reply src/arfs/arfsdao.test.ts]: Included in fab920d

File: src/arfs/arfsdao.test.ts (hunk 2 above)
Comment (fedellen) [reply src/arfs/arfsdao.test.ts:46]: Added: fab920d

**Review by arielmelendez — APPROVED** [review]

**Pull request merged**